src/algebra/lie/basic.lean

```diff
@@ -553,7 +555,7 @@ variables [comm_ring R] [lie_ring L] [lie_algebra R L] [add_comm_group M] [modul
 variables [lie_ring_module L M] [lie_module R L M]
 
 /-- A Lie module yields a Lie algebra morphism into the linear endomorphisms of the module. -/
-def lie_module.to_endo_morphism : L →ₗ⁅R⁆ module.End R M :=
+@[simps] def lie_module.to_endo_morphism : L →ₗ⁅R⁆ module.End R M :=
```

Comment:
I know this predates this PR, but could we remove the underscore in the middle of `endo_morphism`?

Reply:
Done.

For what it's worth, the old name `to_endo_morphism` was supposed to be an abbreviation for `to_endomorphism_morphism` since this is a morphism from the Lie algebra to the endomorphism algebra.